doc/classes/JSONRPC.xml

```diff
@@ -69,11 +69,14 @@
 			<description>
 			</description>
 		</method>
-		<method name="set_scope">
+		<method name="set_method">
 			<return type="void" />
-			<param index="0" name="scope" type="String" />
-			<param index="1" name="target" type="Object" />
+			<param index="0" name="name" type="String" />
+			<param index="1" name="callback" type="Callable" />
 			<description>
+				Registers a callback for the given method name.
+				- [param name] The name, that clients can use to access the callback.
```

Comment:
```suggestion
				- [param name] The name that clients can use to access the callback.
```
Missed this one, this is incorrect comma use in English, it is correct if it's a parenthetical phrase, but not here (i.e. "The method, that is referenced here, is used by clients...")